adam/experiment/observer.py

```diff
@@ -85,6 +88,160 @@ def report(self) -> None:
         )
 
 
+CSS = """
+body {
+    font-size: 1em;
+    font-family: sans-serif;
+}
+
+table td { p
+    padding: 1em; 
+    background-color: #FAE5D3 ;
+}
+"""
+
+
+@attrs(slots=True)
+class HTMLLogger:
```

Comment:
This makes the reader ask: HTML _what_ logger?  Maybe `LearningProgressHtmlLogger`?